src/Draco.Compiler/Internal/Symbols/Metadata/IMetadataClass.cs

```diff
@@ -0,0 +1,6 @@
+namespace Draco.Compiler.Internal.Symbols.Metadata;
+
+internal interface IMetadataClass
+{
+    public string? DefaultMemberAttributeName { get; }
+}
```

Comment:
Please document interface and members.
For the interface, document the fact that this represents something that metadata calls a class, regardless of what we use it for (like static classes are actually modules for our compiler, ...).